src/style.css

```
@@ -1,3 +1,80 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+header {
```

Use only class selector, fix it everywhere 
```suggestion
.header {
```